webExtensionApis/prefs/api.js

```diff
@@ -1,7 +1,10 @@
 "use strict";
 
+/* global ExtensionAPI:false */
+
 ChromeUtils.import("resource://gre/modules/Services.jsm");
 
+/*
```

Comment:
Commented this out, but if they aren't used, we should probably just :snip: :snip: ✂️ 

```
/Users/pdehaan/dev/github/mozilla/shield-studies-addon-utils/webExtensionApis/prefs/api.js
   9:10  error    'get' is defined but never used                no-unused-vars
  24:10  error    'set' is defined but never used                no-unused-vars
```